packages/firebase_auth/README.md

```diff
@@ -83,11 +83,35 @@ _handleSignIn()
     .catchError((e) => print(e));
 ```
 
+### Register a user
+
+```dart
+final FirebaseUser user = await _auth.createUserWithEmailAndPassword(
+      email: 'an email',
+      password: 'a password',
+    );
+```
+
+### Supported Firebase authentication methods
+
+* `Google authentication`
+* `Email&Password authentication`
+* `Phone authentication`
+* `Anonymously Sign in`
+* `Github authentication`
+* `Facebook authentication`
+* `Twitter authentication`
+
 ### Phone Auth
 
 You can use Firebase Authentication to sign in a user by sending an SMS message to
 the user's phone. The user signs in using a one-time code contained in the SMS message.
 
+### After authentication
+
+After a successful authentication, you will receive a `FirebaseUser` object. You can use this object to check if the email is verified, to update email, to send verification email and etc. See [Documentation](https://pub.dartlang.org/documentation/firebase_auth/latest/firebase_auth/FirebaseUser-class.html) for more details on the `FirebaseUser` Object.
```

Comment:
Change etc to "and so on"

I would say something like this:

See the [`FirebaseUser`](https://pub.dartlang.org/documentation/firebase_auth/latest/firebase_auth/FirebaseUser-class.html) API documentation for more details.